vuepress/docs/next/docs/getting-started/development-concepts.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebarDepth: 2
+---
+
+# Development Concepts on Entando
+ 
+On Entando Composition Platform, the development of an application is broken down into modular steps of create, curate, compose, and consume. In the simplest terms, you start by **creating** functional building blocks, **curate** and organize the blocks in a catalogue, **compose** them on a page or website, which is then **consumed** in an app where it can be scaled and improved upon. 
+
+Entando provides a composable environment where these building blocks can be built independently, enabling parallel teams to accelerate development and innovation on a cloud and Kubernetes-native stack.
+
+![Entando Platform Development Process](./img/concepts.png)
+
+## Create Components 
+The first step in building a modern application on Entando is to design and create each independant component or functional block for your application. Developers can use Entando Component Generator or Blueprint to automate the building and packaging of project files, with the flexibility to customize. In this way, new business capabilities are integrated into your application without overhauling existing systems. 
```

Comment:
"the" Entando Component Generator
"are integrated" -> "can be integrated"
"overhauling" -> "rewriting"